src/main/java/aquality/selenium/core/elements/interfaces/IElement.java

```diff
@@ -0,0 +1,38 @@
+package aquality.selenium.core.elements.interfaces;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.remote.RemoteWebElement;
+
+public interface IElement extends IParent {
+
+    /**
+     * Gets clear WebElement.
+     *
+     * @return WebElement
+     */
+    default RemoteWebElement getElement() {
+        return getElement(null);
```

Comment:
what does initializing with null mean?

Reply:
it means that the timeout wasn't specified and we will use the default one